ironic-deployment/ironic/ironic.yaml

```diff
@@ -96,6 +164,24 @@ spec:
         - name: ironic-inspector
           image: quay.io/metal3-io/ironic
           imagePullPolicy: Always
+          readinessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /v1
+              port: 5050
```

Comment:
The 5050 port is the reverse proxy port of ironic-inspector. This reverse proxy runs inside ironic-api, so I think it is not necessary to check this port (because you checked port 6385). The port of Ironic inspector is 5049

Reply:
Each container should have probs, requirement.

Reply:
Do you think we need liveness/readiness probes for the following ?
```
ironic-log-watch
ironic-inspector-log-watch
ironic-endpoint-keepalived
```
For me, they do not seem to be services.

Reply:
@kashifest do you see any need of adding the probes for the above three non-service containers

Reply:
@Xenwar Yes, the ironic-api container open some ports: 80, 6385, 5050. Therefore, if you only need to check if this container running, I think pinging one port is enough.

> ironic-log-watch
> ironic-inspector-log-watch
> ironic-endpoint-keepalived

Sorry I have no answer for this question.

Reply:
Thansk @namnx228

Reply:
btw, this port should be 5049, actually

Reply:
Shouldn't this be port 5049, considering @namnx228 's comment ?

Reply:
Thanks, done.